### ISSUE TITLE
ImportError: No module named 'subbrute'

### ISSUE THREAD

**Issue by pandalatte** [rvrsh3ll/FindFrontableDomains#10]
Hi everyone. Trying to install this thing...

`root@localhost:~/tools/FindFrontableDomains# ./setup.sh 

Collecting argparse
Collecting dnspython
Collecting pyfiglet
Collecting termcolor`

OK, no errors.

Trying to run:

`root@localhost:~/tools/FindFrontableDomains# ./FindFrontableDomains.py 

Traceback (most recent call last):
  File "./FindFrontableDomains.py", line 9, in <module> from Sublist3r import sublist3r
  File "/root/tools/FindFrontableDomains/Sublist3r/sublist3r.py", line 21, in <module> from subbrute import subbrute
ImportError: No module named 'subbrute'`

:( What is missing?

**Comment by nm17** [comment]
You forgot to run setup.sh

**Comment by nm17** [comment]
Is Sublist3r in your directory?

**Comment by pandalatte** [comment]
I did not forget to run setup.sh, its mentioned in my original post.

Yes, sublister is present. It has his own setup.py and requirements.txt but both of them dont say anything about "subbrute" module. I tried "pip install subbrute", and Googling it, but no luck.

root@localhost:~/tools/FindFrontableDomains# pip install subbrute
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
ERROR: Could not find a version that satisfies the requirement subbrute (from versions: none)
ERROR: No matching distribution found for subbrute

drwxr-xr-x  7 root root 4096 Dec  8 18:44 .
drwxr-xr-x 18 root root 4096 Dec  8 18:42 ..
drwxr-xr-x  4 root root 4096 Dec  8 18:44 build
drwxr-xr-x  2 root root 4096 Dec  8 18:44 dist
-rwxr-xr-x  1 root root 5046 Dec  8 18:42 FindFrontableDomains.py
drwxr-xr-x  8 root root 4096 Dec  8 18:42 .git
-rw-r--r--  1 root root 1512 Dec  8 18:42 LICENSE
-rw-r--r--  1 root root  393 Dec  8 18:42 README.md
-rw-r--r--  1 root root   55 Dec  8 18:42 requirements.txt
-rwxr-xr-x  1 root root  326 Dec  8 18:42 setup.sh
drwxr-xr-x  8 root root 4096 Dec  8 18:58 Sublist3r
drwxr-xr-x  2 root root 4096 Dec  8 18:44 Sublist3r.egg-info
